### PR TITLE
Switch to FD.o SDK 19.08

### PR DESCRIPTION
**Pull request by gasinvein**
FD.o SDK 19.08 beta is available now. We need it for multiarch builds.

### CODE REVIEW

**Comment by gasinvein** [comment]
@nedrichards Is the flathub-beta remote available for builds?

**Comment by LeandroStanger** [comment]
bot, build net.lutris.Lutris

File: net.lutris.Lutris.yml [hunk 1]
```diff
@@ -33,17 +33,17 @@ finish-args:
...
     directory: lib/debug/lib/i386-linux-gnu
-    version: "18.08"
+    version: "19.08beta"
     no-autodownload: true
 
   org.freedesktop.Platform.GL32:
     directory: lib/i386-linux-gnu/GL
     version: "1.4"
-    versions: "18.08;1.4"
+    versions: "19.08beta;1.4"
```
Comment (LeandroStanger) [inline net.lutris.Lutris.yml]: 1.4 is still needed?

**Comment by gasinvein** [comment]
Yes, it's the default branch for current Nvidia GL extensions.

File: net.lutris.Lutris.yml (hunk 1 above)
Comment (gasinvein) [reply net.lutris.Lutris.yml]: Yes, it's the default branch for current Nvidia GL extensions.

**Comment by gasinvein** [comment]
bot, build

**Pull request merged**